File: example/java/jimmer-sql-graphql/src/main/java/org/babyfish/jimmer/sql/example/graphql/repository/BookStoreRepository.java
package org.babyfish.jimmer.sql.example.graphql.repository;

import org.babyfish.jimmer.spring.repository.JRepository;
import org.babyfish.jimmer.sql.example.graphql.entities.BookStore;
import org.springframework.lang.Nullable;

import java.util.List;

public interface BookStoreRepository extends JRepository<BookStore, Long> {

    List<BookStore> findByNameLikeOrderByName(@Nullable String name);
}

File: example/kotlin/jimmer-sql-graphql-kt/src/main/kotlin/org/babyfish/jimmer/example/kt/graphql/dal/BookStoreRepository.kt
package org.babyfish.jimmer.example.kt.graphql.dal

import org.babyfish.jimmer.example.kt.graphql.entities.*
import org.babyfish.jimmer.spring.repository.KRepository

interface BookStoreRepository : KRepository<BookStore, Long> {

    fun findByNameLikeOrderByName(name: String?): List<BookStore>
}